card/mod/05_standard/spec/set/type/list_spec.rb

```diff
@@ -0,0 +1,129 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Type::List do
+  subject { Card.fetch('Parry Hotter+authors').item_names.sort }
+  before do
+    Card::Auth.as_bot do
+      Card.create! name: 'Stam Broker+books', type: 'listed by'
+      Card.create!(
+        name: 'Parry Hotter+authors',
+        content: "[[Darles Chickens]]\n[[Stam Broker]]",
+        type: 'list'
+      )
+    end
+  end
+  describe 'Parry Hotter+authors' do
+    context "when 'Parry Hotter' is added to Joe-Ann Rolwings's books" do
+      before do
+        Card.create! name: 'Joe-Ann Rolwing', type: 'author'
+        Card.create!(
+          name: 'Joe-Ann Rolwing+books', type: 'listed by',
+          content: '[[Parry Hotter]]'
+        )
+      end
+      it do
+        is_expected.to eq(
+          ['Darles Chickens', 'Joe-Ann Rolwing', 'Stam Broker']
+        )
+      end
+    end
+
+    context "when 'Parry Hotter' is dropped from Stam Brokers's books" do
+      before do
+        Card::Auth.as_bot do
+          Card['Stam Brokers+books'].update_attributes! :content=>"[[50 grades of shy]]"
```

Comment:
Line is too long. [88/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.